### PR TITLE
changed Resume Builder to Lambda-CRP

### PR DESCRIPTION
**Pull request by BrandonJAllison**
Rebranding from Lambda Resume Builder to Career Readiness Portal

### CODE REVIEW

File: lambda-resume-builder/public/index.html [hunk 1]
```diff
@@ -20,7 +20,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-  <title>Lambda Resume Builder</title>
+  <title>Lambda: CRP</title>
```
Comment (brellin) [inline lambda-resume-builder/public/index.html:23]: 👍

File: lambda-resume-builder/src/Components/Dashboard.js [hunk 2]
```diff
@@ -139,7 +139,7 @@ class Dashboard extends React.Component {
               noWrap
               className={classes.title}
             >
-              Resume Builder
+              Career Readiness Portal Dashboard
```
Comment (brellin) [inline lambda-resume-builder/src/Components/Dashboard.js:142]: # Good name!

File: lambda-resume-builder/src/Components/LogIn/LogIn.js [hunk 3]
```diff
@@ -130,7 +143,8 @@ function SignIn(props) {
 const mapStateToProps = state => (
     
   {
-    login: state.login
+    login: state.login,
+    loggedIn: state.loggedIn
```
Comment (brellin) [inline lambda-resume-builder/src/Components/LogIn/LogIn.js:147]: ## `loggedIn` isn't being used anywhere. It seems like you can probably remove it.

File: lambda-resume-builder/src/Components/Register/Register.js [hunk 4]
```diff
@@ -85,9 +85,17 @@ const Register = (props) =>  {
     setState({...state, [event.target.name]:event.target.value});
   }
 
+  const redirect = () => {
+    console.log('Props', props)
+    window.location.pathname = '/dashboard';
```
Comment (brellin) [inline lambda-resume-builder/src/Components/Register/Register.js:90]: ### Direct manipulation of the `pathname` is a start, but I think we can find a better way eventually.

File: lambda-resume-builder/src/Components/listItems.js [hunk 5]
```diff
@@ -53,13 +53,13 @@ export const secondaryListItems = (
       <ListItemIcon>
         <ShareIcon />
       </ListItemIcon>
-      <ListItemText primary="Share" />
+      <ListItemText primary="Poop" />
```
Comment (brellin) [inline lambda-resume-builder/src/Components/listItems.js:56]: # Professional! 💩

File: lambda-resume-builder/src/Reducers/rootReducer.js [hunk 6]
```diff
@@ -10,6 +10,7 @@ const initialState = {
     
     savingUser: false,
     loggingIn: false,
+    loggedIn: false
```
Comment (brellin) [inline lambda-resume-builder/src/Reducers/rootReducer.js:13]: ## Pointless, again.

**Review by brellin — COMMENTED** [review]
# I give this my stamp ㊙️ of approval 👍

File: lambda-resume-builder/src/Components/LogIn/LogIn.js (hunk 3 above)
Comment (BrandonJAllison) [reply lambda-resume-builder/src/Components/LogIn/LogIn.js:147]: I think it was you who made me create it!

**Pull request merged**